contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.0.0--3.1.0.sql

```diff
@@ -31,6 +31,37 @@ end
 $$
 LANGUAGE plpgsql;
 
+CREATE OR REPLACE FUNCTION sys.babelfish_update_server_collation_name() RETURNS VOID
+LANGUAGE C
+AS 'babelfishpg_common', 'babelfish_update_server_collation_name';
+
+DO
+LANGUAGE plpgsql
+$$
+BEGIN
+    -- Check if the GUC is empty
+    IF current_setting('babelfishpg_tsql.restored_server_collation_name', true) <> '' 
+        OR current_setting('babelfishpg_tsql.restored_server_collation_name', true) IS NOT NULL THEN
```

Comment:
This should be `AND` right? Also, we should first check if it is NOT NULL and then check if it not empty.

Reply:
I have updated the condition in latest rev. As discussed offline, NULL check is not required, check for empty string is sufficient.